Análise da representatividade do perfil dos candidatos do Enem 2019

Rafael Augusto Bonin Bisoffi

contato: rafael_bisoffi@hotmail.com

1. Introdução

Segundo o site Brasil Escola (2018), o Enem (abreviação de Exame Nacional do Ensino Médio) surgiu em 1998 com o propósito inicial de avaliar os alunos do Ensino Médio. A prova não tinha a magnitude que tem hoje: agora já se trata da maior prova nacional, sendo aceita para ingresso na maior parte das universidades públicas, em muitas privadas e até em algumas internacionais (BRASIL ESCOLA, 2018). Seu crescimento se deu gradualmente e associado a diversos outros projetos, como o ProUni (Programa Universidade para Todos) e o SiSU (Sistema de Seleção Unificada).

2. Objetivos

O objetivo da presente análise é verificar se o perfil dos candidatos ao Enem de 2019 é compatível com dados da sociedade brasileira em geral: trata-se, portanto, de uma questão de representatividade. Se os dados apontarem para uma discrepância, isso é indicativo da necessidade de atenção do setor público para certos setores da sociedade que seriam excluídos do acesso a essa prova tão importante. Os indicadores escolhidos foram de gênero, raça e tipo de escola (pública X privada) – os quais podem apontar para processos históricos de exclusão motivados por problemas da sociedade brasileira, como racismo estrutural, misoginia e elitismo.


3. Descrição geral dos dados

Os dados utilizados foram disponibilizados pelo INEP (Instituto Nacional de Estudos e Pesquisas Educacionais), órgão responsável pelo Enem, em seu site (INEP, 2021). Esses dados incluem uma quantidade gigantesca de informações. Segundo o próprio INEP, “Os microdados do Enem são o menor nível de desagregação de dados recolhidos por meio do exame. Eles atendem a demanda por informações específicas ao disponibilizar as provas, os gabaritos, as informações sobre os itens, as notas e o questionário respondido pelos inscritos no Enem” (INEP, 2021). Sua disponibilização gratuita e aberta para pesquisadores cumpre um papel importante de transparência de informações do governo e pode contribuir para melhorias importantes a partir de pesquisa. O arquivo específico que foi consultado é o intitulado como “MICRODADOS_ENEM_2019”, em formato .csv. Abaixo pode-se ver as primeiras linhas do arquivo, de modo a se ter uma ideia do tipo de informação que traz.

In [3]:
import numpy as np
import pandas as pd

enem_2019 = pd.read_csv(r'C:\Users\Rafael\Desktop\Semantix\DADOS\MICRODADOS_ENEM_2019.csv', sep=';', encoding='latin-1')
enem_2019.head(10)

NU_INSCRICAO  NU_ANO  CO_MUNICIPIO_RESIDENCIA NO_MUNICIPIO_RESIDENCIA  \
0  190001595656    2019                  3552205                Sorocaba   
1  190001421546    2019                  2910800        Feira de Santana   
2  190001133210    2019                  2304400               Fortaleza   
3  190001199383    2019                  1721000                  Palmas   
4  190001237802    2019                  3118601                Contagem   
5  190001782198    2019                  5107602            Rondonópolis   
6  190001421548    2019                  2924009            Paulo Afonso   
7  190001595657    2019                  3550308               São Paulo   
8  190001592264    2019                  3550308               São Paulo   
9  190001592266    2019                  3550308               São Paulo   

   CO_UF_RESIDENCIA SG_UF_RESIDENCIA  NU_IDADE TP_SEXO  TP_ESTADO_CIVIL  \
0                35               SP      36.0       M                1   
1                29               BA      23.0       M                1   
2                23               CE      39.0       F                1   
3                17               TO      25.0       F                1   
4                31               MG      22.0       F                1   
5                51               MT      37.0       M                2   
6                29               BA      22.0       F                1   
7                35               SP      20.0       M                1   
8                35               SP      20.0       F                1   
9                35               SP      17.0       M                1   

   TP_COR_RACA  ...  Q016  Q017 Q018  Q019 Q020  Q021  Q022  Q023  Q024  Q025  
0            3  ...     A     A    A     A    A     A     C     A     C     B  
1            1  ...     A     A    A     B    A     A     E     A     A     B  
2            3  ...     A     A    A     B    A     A     C     A     A     B  
3            1  ...     A     A    A     C    A     B     D     A     B     B  
4            1  ...     B     A    A     B    A     A     C     B     B     B  
5            2  ...     B     A    A     B    A     A     E     A     B     B  
6            3  ...     A     A    A     B    A     A     B     A     A     A  
7            3  ...     B     A    A     B    A     A     C     A     A     B  
8            1  ...     B     A    A     B    A     A     B     B     A     A  
9            1  ...     B     A    A     B    A     B     C     B     B     B  

[10 rows x 136 columns]

Para contraponto, utilizaram-se dois censos: a Pesquisa Nacional por Amostra de Domicílios (PNAD) 2019, realizada pelo IBGE (nas referências, IBGE 2019a e 2019b) e o Censo Escolar do Ministério da Educação (MEC, 2019). Esses levantamentos de dados da sociedade brasileira em geral são essenciais para que se possa avaliar se, de fato, o perfil dos candidatos ao Enem 2019 foi representativo das diversas parcelas que compõem a sociedade brasileira.

4. Gênero

In [5]:
total_masculino = (enem_2019.TP_SEXO == 'M').sum()
total_feminino = (enem_2019.TP_SEXO == 'F').sum()
total_genero = total_masculino + total_feminino

porcentagem_masculino = total_masculino/total_genero
porcentagem_feminino = total_feminino/total_genero

print("A porcentagem de candidatos homens é: " + str(round(porcentagem_masculino, 2)))
print("A porcentagem de candidatas mulheres é: " + str(round(porcentagem_feminino, 2)))


A porcentagem de candidatos homens é: 0.4
A porcentagem de candidatas mulheres é: 0.6


Os dados de gênero, surpreendentemente, não apontam apenas para uma igualdade entre os sexos, mas, em realidade, para uma "vantagem" das mulheres, que comporiam aproximadamente 60% dos candidatos, contra quase 52% da população em geral (IBGE, 2019a). Na PNAD (IBGE, 2019a), os homens são, de fato, minoria: em torno de 48% da população em geral; mas, entre os candidatos do Enem 2019, eles caem para 40%. Contudo, recomenda-se cautela na interpretação deste dado. Em primeiro lugar, porque os dados representam a resposta a um formulário no momento da inscrição – desconsidera, portanto, dados de absenteísmo, que podem eventualmente apontar para desigualdades. Em segundo lugar, tal dado desconsidera o tipo de curso escolhido: é patente que há cursos superiores historicamente considerados “femininos” – como os relativos a cuidados e à educação básica, como enfermagem e pedagogia, e cursos historicamente considerados “masculinos”, sobretudo os de ciências exatas, como engenharias, e outros prestigiosos, como medicina e direito. Um eventual cruzamento desses dados poderia matizar a interpretação deste indicador e apontar para desigualdades significativas, como destaca o trabalho de Pereira e Favaro (2017): "No caso da pesquisa realizada na Unespar, Campus de Paranavaí, constatou-se que as mulheres são maioria no ensino superior atualmente, embora os cursos com maior predominância delas sejam de fato aqueles considerados como tipicamente femininos ... Outro ponto a ser considerado é que a maioria das mulheres encontra-se atualmente em postos de trabalho com baixa remuneração, como os de serviços, nas áreas da saúde e da educação ... Quanto à escolha do curso ou da carreira pela mulher, muitas vezes a opção é diferente da desejada, por falta de condições econômicas, ou seja, em consequência de sua realidade material, e também pela baixa condição escolar que possuem, e não necessariamente por vocação ou devido a uma suposta falsa consciência sobre sua posição profissional diante da sociedade".

5. Raça

In [6]:
total_pardos = (enem_2019.TP_COR_RACA == 3).sum()
total_brancos = (enem_2019.TP_COR_RACA == 1).sum()
total_pretos = (enem_2019.TP_COR_RACA == 2).sum()
total_amarelos = (enem_2019.TP_COR_RACA == 4).sum()
total_indigenas = (enem_2019.TP_COR_RACA == 5).sum()
total_raca = total_pardos + total_brancos + total_pretos + total_amarelos + total_indigenas

porcentagem_pardos = total_pardos/total_raca
porcentagem_brancos = total_brancos/total_raca
porcentagem_pretos = total_pretos/total_raca
porcentagem_amarelos = total_amarelos/total_raca
porcentagem_indigenas = total_indigenas/total_raca

print("A porcentagem de candidatos pardos é: " + str(round(porcentagem_pardos, 2)))
print("A porcentagem de candidatos brancos é: " + str(round(porcentagem_brancos, 2)))
print("A porcentagem de candidatos pretos é: " + str(round(porcentagem_pretos, 2)))
print("A porcentagem de candidatos amarelos é: " + str(round(porcentagem_amarelos, 2)))
print("A porcentagem de candidatos indígenas é: " + str(round(porcentagem_indigenas, 3)))

A porcentagem de candidatos pardos é: 0.47
A porcentagem de candidatos brancos é: 0.37
A porcentagem de candidatos pretos é: 0.13
A porcentagem de candidatos amarelos é: 0.02
A porcentagem de candidatos indígenas é: 0.006


Os dados de raça também surpreendem. Segundo a PNAD (IBGE, 2019b), os dados de raça no Brasil, em geral, colhidos por autodeclaração, se distribuem da seguinte forma: 42,7 se declaram brancos; 46,8, pardos; 9,4, pretos; e 1,1% se dividem entre amarelos e indígenas. Nesse sentido, os dados do INEP mostram um perfil racial também bastante diverso no Enem 2019, inclusive com maior representatividade de pardos e pretos, que somariam 60% dos candidatos (contra aproximadamente 56% da população geral, segundo o PNAD). Os candidatos que se autodeclararam brancos na inscrição do INEP contam 37%, contra os quase 43% encontrados na PNAD. Indígenas e amarelos somam uma minoria, como também encontrado no PNAD. Aqui é necessária uma grande cautela ao interpretar os dados. A questão racial no Brasil é extremamente complexa e multifacetada. Primeiro, poderíamos questionar, para análise, a autodeclaração, sobretudo quando consideramos que, para o acesso no Ensino Superior, com a política de cotas, há um certo incentivo para a autodeclaração como não-branco, o que não se encontra numa pesquisa genérica como a PNAD. Em especial a categoria "pardo", ainda que oficialmente utilizada, gera bastantes questionamentos por diversos movimentos sociais e pode criar imprecisões, como aponta Silveira (2019). O recorte de idade também pode ser um fator importante. Novamente, é preciso ressaltar que os dados indicam o perfil dos inscritos, mas não o dos que realizaram efetivamente a prova. Também desconsideram, portanto, desempenho. Igualmente não reconhecem os cursos escolhidos - há cursos historicamente mais "brancos", sobretudo os muito prestigiosos, como medicina – que, segundo o site Virando Bixo (2019), é o curso com menor inclusão racial. Essas ressalvas podem indicar grades desigualdades se investigadas mais a fundo.

6. Tipo de escola

In [9]:
total_publica = (enem_2019.TP_ESCOLA == 2).sum()
total_privada = (enem_2019.TP_ESCOLA == 3).sum()
total_escolas_resp = total_publica + total_privada
total_no_resp = (enem_2019.TP_ESCOLA == 1).sum()
total_escolas = total_escolas_resp + total_no_resp

porcentagem_publica = total_publica/total_escolas_resp
porcentagem_privada = total_privada/total_escolas_resp
porcentagem_no_resp = total_no_resp/total_escolas

print("Dentre os que marcaram, a porcentagem de candidatos de escola pública é: " + str(round(porcentagem_publica, 2)))
print("Dentre os que marcaram, a porcentagem de candidatos de escola privada é: " + str(round(porcentagem_privada, 2)))
print("Porém, uma grande porcentagem escolheu não marcar: " + str(round(porcentagem_no_resp, 2)))

Dentre os que marcaram, a porcentagem de candidatos de escola pública é: 0.85
Dentre os que marcaram, a porcentagem de candidatos de escola privada é: 0.15
Porém, uma grande porcentagem escolheu não marcar: 0.71


Segundo o Censo Escolar realizado pelo MEC em 2019, 87% dos alunos do Ensino Médio (fase do ensino crítica para o ingresso no Ensino Superior) estão matriculados em escolas públicas (entre estaduais e federais), enquanto 12,5% em escolas particulares (MEC, 2019). Esse dado pode ser facilmente visualizado no infográfico publicado pelo MEC: https://download.inep.gov.br/educacao_basica/censo_escolar/download/2019/infografico_censo_2019.pdf. Nesse sentido, os dados do INEP apontam uma boa representatividade dos tipos de ensino entre os candidatos do Enem 2019, pois revelam que 85% dos candidatos se declaram como alunos de escola pública, enquanto 15% se declaram de escola particular. Contudo, os dados referidos consideram apenas os candidatos que marcaram alguma opção – infelizmente, uma grande parcela optou por não o fazer: 71% do total não declarou seu tipo de escola, o que levanta um questionamento do motivo para isso. Seria enventualmente interessante que o INEP tornasse obrigatória essa declaração para que se obtenham dados mais completos. Cabem aqui também ressalvas feitas acima: os dados escolhidos não indicam se o candidato realizou efetivamente a prova, qual curso o candidato escolheu e qual seu desempenho – cruzamentos que podem revelar desigualdades. No entanto, a grande representatividade da escola pública ao menos no ato de inscrição aponta, de fato, para um efeito positivo de certas políticas, como a isenção da taxa de inscrição para alunos do sistema público, estabelecida em 2001 (BRASIL ESCOLA, 2018), o que certamente contribuiu para essa representatividade.

7. Conclusões

Os indicadores escolhidos apresentam, em primeira análise, que há uma boa representatividade quando se comparam o perfil dos inscritos no Enem de 2019 e o dados da sociedade brasileira em geral obtidos pelos censos citados. Isso significaria que as políticas relativas à prova têm obtido sucesso em garantir que diversos setores da sociedade brasileira tenham acesso à inscrição da prova, o que apontaria para uma recomendação de manter tais políticas, como a de taxa de isenção, que provavelmente tem um efeito de facilitação do acesso à prova para alunos de renda baixa, por exemplo.

É claro que esta é uma análise limitada, que poderia ser expandida em diversas direções. Os dados analisados, por exemplo, não levam em conta o desempenho do candidato, nem se todos tiveram condições de realmente fazer as provas, nem qual o curso escolhido (cruzamentos que poderiam indicar divergências importantes). Um exemplo: seria ideal cruzar os dados levantados com o desempenho dos candidatos, o que poderia apontar para desigualdades significativas, como entre alunos de escolas particulares e públicas, que demandariam atenção especial do poder público.

Outro ponto importante a se considerar é que este foi o último Enem a ser aplicado antes da pandemia de Covid-19. Também que 2019 foi o primeiro ano do governo de Jair Bolsonaro, portanto, um ano de “aclimatação”, que provavelmente não implicou em mudanças significativas nas políticas públicas relativas às provas. Já há indícios de mudanças negativas nos indicadores aqui escolhidos quando se trata as provas de 2020 e 2021 (G1, 2021), cujos microdados ainda não foram disponibilizados. Um passo importante seria cruzar os dados relativos a estes três últimos anos da prova, o que pintaria um quadro evidentemente insondável exclusivamente com os dados de 2019.


8. Referências

BRASIL ESCOLA. Enem 20 anos: a transformação da maior prova do Brasil. (2018). Disponível em: https://vestibular.brasilescola.uol.com.br/enem/enem-20-anos-transformacao-maior-prova-brasil.htm. Acesso: 07 dez. 2021.

G1. Enem 2021: número de pretos, pardos e indígenas inscritos cai mais de 50%. (2021). Disponível em: https://g1.globo.com/educacao/enem/2021/noticia/2021/08/27/enem-2021-cai-negros-pardos-indigenas-inscritos.ghtml. Acesso: 08 dez. 2021.
                
IBGE. Quantidade de homens e mulheres. (2019a). Disponível em: https://educa.ibge.gov.br/jovens/conheca-o-brasil/populacao/18320-quantidade-de-homens-e-mulheres.html. Acesso: 07 dez. 2021.
    
IBGE. Cor ou raça. (2019b). Disponível em: https://educa.ibge.gov.br/jovens/conheca-o-brasil/populacao/18319-cor-ou-raca.html. Acesso: 07 dez. 2021.
    
INEP. Microdados Enem 2019. (2021). Disponível em: https://www.gov.br/inep/pt-br/acesso-a-informacao/dados-abertos/microdados/enem. Acesso: 07 dez. 2021.
            
MEC. Censo Escolar 2019. (2019). Disponível em: http://portal.mec.gov.br/component/tags/tag/censo-escolar. Acesso: 07 dez. 2021.

PEREIRA, Ana Cristina Furtado; FAVARO, Neide de Almeida Lança Galvão. História da mulher no ensino superior e suas condições atuais de acesso e permancência. EDUCERE - XV Congresso Nacional de Educação, 2017. Disponível em: https://educere.bruc.com.br/arquivo/pdf2017/26207_12709.pdf. Acesso: 08 dez. 2021.

SILVEIRA, Marcos Silva de. Problemas relacionados a noção de “pardo” como categoria identitária no Brasil. VI Encontro de Administração Pública. Salvador. Junho de 2019. Disponível no link: https://ebap.online/ebap/index.php/ebap/viebap/paper/viewFile/543/251. Acesso: 08 dez. 2021.

VIRANDO BIXO. Medicina é a graduação com menor inclusão de negros. (2019). Disponível em: https://www.virandobixo.com.br/NOT,0,0,1463817,medicina+e+a+graduacao+com+a+menor+inclusao+de+negros.aspx. Acesso: 08 dez. 2021.
